@xen-orchestra/web/src/components/backups/configuration/BackupJobSchedules.vue

```diff
@@ -0,0 +1,226 @@
+                  <span class="visually-hidden">{{ t('last-run-number', { n: index + 1 }) }}</span>
+                </li>
+              </ul>
+            </template>
+          </td>
+        </tr>
+      </template>
+    </VtsDataTable>
+    <UiTopBottomTable :selected-items="0" :total-items="0">
+      <UiTablePagination v-bind="paginationBindings" />
+    </UiTopBottomTable>
+  </UiCard>
+</template>
+
+<script setup lang="ts">
+import type { XoBackupJob } from '@/remote-resources/use-xo-backup-job-collection.ts'
+import { useXoBackupLogCollection } from '@/remote-resources/use-xo-backup-log-collection'
+import { useXoScheduleCollection } from '@/remote-resources/use-xo-schedule-collection'
+import type { XoBackupLog } from '@/types/xo/backup-log.type'
+import type { IconName } from '@core/icons'
+import VtsDataTable from '@core/components/data-table/VtsDataTable.vue'
+import VtsEnabledState from '@core/components/enabled-state/VtsEnabledState.vue'
+import VtsIcon from '@core/components/icon/VtsIcon.vue'
+import UiButtonIcon from '@core/components/ui/button-icon/UiButtonIcon.vue'
+import UiCard from '@core/components/ui/card/UiCard.vue'
+import UiCheckbox from '@core/components/ui/checkbox/UiCheckbox.vue'
+import UiLink from '@core/components/ui/link/UiLink.vue'
+import UiQuerySearchBar from '@core/components/ui/query-search-bar/UiQuerySearchBar.vue'
+import UiTablePagination from '@core/components/ui/table-pagination/UiTablePagination.vue'
+import UiTitle from '@core/components/ui/title/UiTitle.vue'
+import UiTopBottomTable from '@core/components/ui/top-bottom-table/UiTopBottomTable.vue'
+import { usePagination } from '@core/composables/pagination.composable'
+import { useRouteQuery } from '@core/composables/route-query.composable'
+import { useTable } from '@core/composables/table.composable'
+import { vTooltip } from '@core/directives/tooltip.directive'
+import { createMapper } from '@core/packages/mapper'
+import { noop } from '@vueuse/shared'
+import { computed, ref } from 'vue'
+import { useI18n } from 'vue-i18n'
+
+const { backupJobs } = defineProps<{
+  backupJobs: XoBackupJob[]
+}>()
+
+const { t } = useI18n()
+
+const { getLastNBackupLogsByJobId } = useXoBackupLogCollection()
+const scheduleData = useXoScheduleCollection()
+const selectedBackupJobId = useRouteQuery('id')
+
+const searchQuery = ref('')
+
+const filteredBackupJobs = computed(() => {
+  const searchTerm = searchQuery.value.trim().toLocaleLowerCase()
+
+  let filteredJob: XoBackupJob[]
+  if (!searchTerm) {
+    filteredJob = backupJobs
+  }
+
+  filteredJob = backupJobs.filter(backupJob =>
+    Object.values(backupJob).some(value => String(value).toLocaleLowerCase().includes(searchTerm))
+  )
+
+  const schedule = filteredJob
+    .map(job => {
+      return (scheduleData.schedulesByJobId.value.get(job.id) ?? []).map(schedule => {
+        return { schedule, job }
+      })
+    })
+    .flat()
+
+  return schedule
+})
```

Comment:
Your filter doesn't work, I think something like that will work : 

```suggestion
const filteredBackupJobs = computed(() => {
  const searchTerm = searchQuery.value.trim().toLocaleLowerCase()

  const filteredJobsSchedule = backupJobs.flatMap(job => {
    return (scheduleData.schedulesByJobId.value.get(job.id) ?? []).map(schedule => {
      return { schedule, job }
    })
  })

  if (!searchTerm) {
    return filteredJobsSchedule
  }

  return filteredJobsSchedule.filter(({ schedule }) =>
    Object.values(schedule).some(value => String(value).toLocaleLowerCase().includes(searchTerm))
  )
})
```